lib/matplotlib/tests/test_datetime.py

```diff
@@ -399,11 +399,20 @@ def test_matshow(self):
         fig, ax = plt.subplots()
         ax.matshow(...)
 
-    @pytest.mark.xfail(reason="Test for pcolor not written yet")
     @mpl.style.context("default")
     def test_pcolor(self):
-        fig, ax = plt.subplots()
-        ax.pcolor(...)
+        base_date = datetime.date(2020, 1, 1)
+        dates = [base_date + datetime.timedelta(days=i) for i in range(10)]
+        data = np.random.rand(10, 10)
```

Comment:
If you want to use randomness, you will need to set a seed (we use 19680801 for tests) or else the test image will change on each run.